components/AssetFormSellPopupMenu/index.tsx

```diff
@@ -7,26 +7,35 @@ import {
 } from './AssetFormSellPopupMenu.styled';
 import { GradientBackground } from '../NavBar/NavBar.styled';
 import { ReactComponent as Ellipsis } from '../../public/ellipsis.svg';
-import { useModalContext, MODAL_TYPES } from '../Provider';
+import {
+  useModalContext,
+  MODAL_TYPES,
+  CreateMultipleSalesModalProps,
+  CancelMultipleSalesModalProps,
+} from '../Provider';
 import { useScrollLock, useEscapeKeyClose } from '../../hooks';
 
 const AssetFormSellPopupMenu = (): JSX.Element => {
-  const { openModal } = useModalContext();
+  const { openModal, modalProps } = useModalContext();
   const [isOpen, setIsOpen] = useState<boolean>(false);
   const togglePopupMenu = () => setIsOpen(!isOpen);
   const closePopupMenu = () => setIsOpen(false);
+  const { saleIds } = modalProps as CancelMultipleSalesModalProps;
+  const { assetIds } = modalProps as CreateMultipleSalesModalProps;
   useScrollLock(isOpen);
   useEscapeKeyClose(closePopupMenu);
 
   const popupMenuItems = [
     {
+      type: 'Sell',
       name: 'Mark all for sale',
       onClick: () => {
         setIsOpen(false);
         openModal(MODAL_TYPES.CREATE_MULTIPLE_SALES);
       },
     },
```

Comment:
Instead of using an `if` statement in the JSX, let's just set up an `isHidden` property for each of the `popupMenuItems`. For example, the `Sell` object could have an `isHidden` property based on whether `assetIds` is an empty array.